deny.toml

```diff
@@ -0,0 +1,30 @@
+targets = [
+    { triple = "x86_64-unknown-linux-gnu" },
+    { triple = "x86_64-unknown-linux-musl" },
+    { triple = "x86_64-apple-darwin" },
+    { triple = "x86_64-pc-windows-msvc" },
```

Comment:
afaik we don't support windows.

Reply:
Okay. I based this off the default cargo-deny deny.toml. I'll remove this.